tembo-operator/src/controller.rs

```diff
@@ -378,23 +383,16 @@ impl CoreDB {
                     last_fully_reconciled_at: None,
                 }
             }
+            // If the cluster is stopped, set the status to a very sparse set
+            // of information that will allow other provisioning teps to
+            // complete, but otherwise do nothing else. The CNPG operator will
+            // handle the rest.
             true => {
-                let current_config_values = get_current_config_values(self, ctx.clone()).await?;
-                CoreDBStatus {
-                    running: false,
-                    extensionsUpdating: false,
-                    storage: Some(self.spec.storage.clone()),
-                    extensions: self.status.as_ref().and_then(|f| f.extensions.clone()),
-                    trunk_installs: self.status.as_ref().and_then(|f| f.trunk_installs.clone()),
-                    resources: Some(self.spec.resources.clone()),
-                    runtime_config: Some(current_config_values),
-                    first_recoverability_time: self
-                        .status
-                        .as_ref()
-                        .and_then(|f| f.first_recoverability_time),
-                    pg_postmaster_start_time: None,
-                    last_fully_reconciled_at: None,
-                }
```

Comment:
I am not really sure why you're removing these.

Reply:
This part I think makes sense. Basically it is leave the status as-is on the K8s resource when you stop the instance, except it's clearing out pg_postmaster_start_time, setting last_fully_reconciled_at, and setting running to false

Reply:
Steve recommended that, and I agreed at the time, but again, I don't know what's actually happening there.